README.md

```diff
@@ -287,47 +287,54 @@ Below you can find a recommended configuration which is based on the [Angular St
-| `i18n`                                 | _Experimental_ |
-| `no-unused-css`                        | _Experimental_ |
-| `template-conditional-complexity`      | _Experimental_ |
-| `angular-whitespace`                   |  _Deprecated_  |
+| Rule                                          |     Status     |
+| --------------------------------------------- | :------------: |
+| `banana-in-box`                               |     Stable     |
+| `contextual-life-cycle`                       |     Stable     |
+| `decorator-not-allowed`                       |     Stable     |
+| `pipe-impure`                                 |     Stable     |
+| `templates-no-negated-async`                  |     Stable     |
+| `no-attribute-parameter-decorator`            |     Stable     |
+| `no-forward-ref`                              |     Stable     |
+| `no-input-prefix`                             |     Stable     |
+| `no-input-rename`                             |     Stable     |
+| `no-output-on-prefix`                         |     Stable     |
+| `no-output-rename`                            |     Stable     |
+| `use-life-cycle-interface`                    |     Stable     |
+| `use-pipe-decorator`                          |     Stable     |
+| `use-pipe-transform-interface`                |     Stable     |
+| `use-view-encapsulation`                      |     Stable     |
+| `component-class-suffix`                      |     Stable     |
+| `component-selector`                          |     Stable     |
+| `directive-class-suffix`                      |     Stable     |
+| `directive-selector`                          |     Stable     |
+| `use-host-property-decorator`                 |     Stable     |
+| `use-input-property-decorator`                |     Stable     |
+| `use-output-property-decorator`               |     Stable     |
+| `trackBy-function`                            |     Stable     |
+| `import-destructuring-spacing`                |     Stable     |
+| `no-output-named-after-standard-event`        |     Stable     |
+| `max-inline-declarations`                     |     Stable     |
+| `prefer-output-readonly`                      |     Stable     |
+| `enforce-component-selector`                  |     Stable     |
+| `no-life-cycle-call`                          |     Stable     |
+| `no-template-call-expression`                 |     Stable     |
+| `no-queries-parameter`                        |     Stable     |
+| `prefer-inline-decorator`                     |     Stable     |
+| `template-cyclomatic-complexity`              |     Stable     |
+| `pipe-prefix`                                 | _Experimental_ |
+| `no-conflicting-life-cycle-hooks`             | _Experimental_ |
+| `i18n`                                        | _Experimental_ |
+| `no-unused-css`                               | _Experimental_ |
+| `template-conditional-complexity`             | _Experimental_ |
```

Comment:
I think we can change the status to `Stable` for this rule;

Reply:
@mgechev can you confirm this?

Reply:
👍

Reply:
Hmm.. there's an open issue relating problems with this rule: #627.

Reply:
@wKoza :point_up_2: